va/libva.def

```diff
@@ -0,0 +1,74 @@
+LIBRARY va
+EXPORTS
+    vaInitialize
+    vaErrorStr
+    vaTerminate
+    vaQueryVendorString
+    vaMaxNumProfiles
+    vaMaxNumEntrypoints
+    vaQueryConfigProfiles
+    vaQueryConfigEntrypoints
+    vaGetConfigAttributes
+    vaProfileStr
+    vaEntrypointStr
+    vaConfigAttribTypeStr
+    vaAcquireBufferHandle
+    vaAssociateSubpicture
+    vaBeginPicture
+    vaCreateBuffer
+    vaCreateConfig
+    vaCreateContext
+    vaCreateImage
+    vaCreateSubpicture
+    vaCreateSurfaces
+    vaDeassociateSubpicture
+    vaDeriveImage
+    vaDestroyBuffer
+    vaDestroyConfig
+    vaDestroyContext
+    vaDestroyImage
+    vaDestroySubpicture
+    vaDestroySurfaces
+    vaEndPicture
+    vaGetDisplayAttributes
+    vaGetImage
+    vaMapBuffer
+    vaMaxNumDisplayAttributes
+    vaMaxNumImageFormats
+    vaMaxNumSubpictureFormats
+    vaPutImage
+    vaQueryDisplayAttributes
+    vaQueryImageFormats
+    vaQuerySubpictureFormats
+    vaQuerySurfaceAttributes
+    vaQuerySurfaceStatus
+    vaQueryVideoProcFilterCaps
+    vaQueryVideoProcFilters
+    vaQueryVideoProcPipelineCaps
+    vaReleaseBufferHandle
+    vaRenderPicture
+    vaSetDisplayAttributes
+    vaSetErrorCallback
+    vaSetInfoCallback
+    vaSetSubpictureGlobalAlpha
+    vaSyncSurface
+    vaUnmapBuffer
+    vaSetDriverName
+    vaSyncBuffer
+    vaQueryConfigAttributes
+    vaBufferSetNumElements
+    vaQuerySurfaceError
+    vaSetImagePalette
+    vaSetSubpictureChromakey
+    vaBufferInfo
+    vaLockSurface
+    vaUnlockSurface
+    vaCreateMFContext
+    vaMFAddContext
+    vaMFReleaseContext
+    vaMFSubmit
+    vaCreateBuffer2
+    vaQueryProcessingRate
+    vaExportSurfaceHandle
+    va_newDisplayContext
+    va_newDriverContext

```

Comment:
Pretty sure that "va_new*" are not supposed to be here. Please double-check against the public header `va.h`

Reply:
I added these two to be exported in va.dll so they can be linked/called from va_win32.dll

Reply:
Right, the libva.so ABI is a mess. I should probably stop complaining and send some patches instead :sweat_smile: 

Please ignore my suggestion.